TypeCobol.LanguageServer/VsCodeProtocol/Initialize Method/InitializeParams.cs

```diff
@@ -26,5 +26,10 @@ public class InitializeParams
         /// The capabilities provided by the client (editor)
         /// </summary>
         public ClientCapabilities capabilities { get; set; }
+
+        /// <summary>
+        /// The initial trace setting. If omitted trace is disabled ('off').
+        /// </summary>
+        public string trace { get; set; }
```

Comment:
Should it be `nullable `to comply with LSP specification?

Reply:
Well it is. The nullable context is deactivated here so this `string` accepts `null` value.